### PR TITLE
Require doctrine/persistence explicitly

### PR DESCRIPTION
**Pull request by franmomu**
## Subject

`doctrine/persistence` is used in [ModelManager](https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/blob/3.x/src/Model/ModelManager.php#L17) but it was never required, I added minimum of 1.3.4 so we can change the namespace.

I added `patch` label because it could remove some warnings.
 

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Added doctrine/persistence as a dependency.
```

### CODE REVIEW

File: src/Model/ModelManager.php [hunk 1]
```diff
@@ -14,9 +14,9 @@
 namespace Sonata\DoctrineMongoDBAdminBundle\Model;
 
 use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\Common\Persistence\Mapping\ClassMetadata as CommonClassMetadata;
 use Doctrine\ODM\MongoDB\Query\Builder;
 use Doctrine\Persistence\Mapping\ClassMetadata;
+use Doctrine\Persistence\Mapping\ClassMetadata as PersistenceClassMetadata;
```
Comment (phansys) [inline src/Model/ModelManager.php:19]: This class is already imported in the previous line.

Comment (franmomu) [reply src/Model/ModelManager.php]: Thanks!

**Review by core23 — APPROVED** [review]

**Comment by phansys** [comment]
If this is a patch, please specify what issues it fixes in the changelog.

**Comment by greg0ire** [comment]
Thanks @franmomu !

**Pull request merged**